appmap/_implementation/event.py

```diff
@@ -70,9 +72,12 @@ def make(fn, isstatic):
         introspecting the given function.
         """
         defined_class, method_id = split_function_name(fn)
+        app_root = os.path.dirname(os.path.realpath(__file__))
```

Comment:
`__file__` is the location of this source file, i.e. the path to `event.py`, which will be a subdirectory under the directory where the `appmap-python` package is installed.

There's no reason for this to be related to the directory where the source file for `fn` is located. Instead, we need to be capturing the absolute path to the current working directory on [startup](https://github.com/applandinc/appmap-python/blob/master/appmap/_implementation/configuration.py#L46), and stripping that from the path.

You'll also need to add at least one test to show that this functionality works.